### PR TITLE
CPP-298 Wait for approval on CircleCI builds from nori and renovate PRs

### PR DESCRIPTION
**Pull request by emortong**
Renovate and Nori create many PRs which queue loads of builds in CircleCI blocking other PRs across the entire organisation from building, as a solution we are having PRs created by Renovate and Nori pause from building until approved. <br/><br/>This PR was created using a nori script ?<br/><br/>_Nori is a command-line application for managing changes across multiple (usually Github) repositories._

### CODE REVIEW

**Comment by emortong** [comment]
Before merging this would need to be removed, I'll just need approval to remove this required check

image

**Review by andygout — COMMENTED** [review]
Is my understanding correct that we need to uncheck `build-test` because for Nori and Renovate branches this will not be run, and therefore such branches will be blocked because it is insisting on a process being run that never will be?

**Comment by emortong** [comment]
@andygout yes. There are also required checks for the jobs `build` and `test` on their own, which are run by the `build-test` worflow and also by the `renovate-nori-build-test` workflow, so I think it is safe to remove that `build-test` required check

**Review by andygout — APPROVED** [review]

**Pull request merged**
